src/style.css

```diff
@@ -1,3 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 12px;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
```

Comment:
Applying `text-transform: uppercase;` to the entire HTML document might not be ideal as it affects all text. Consider applying it only to specific elements where uppercase text is desired.